### PR TITLE
Jmanko/sca upstreaming

### PR DESCRIPTION
**Pull request by jacek-manko-red**

### CODE REVIEW

File: Source/WebCore/platform/graphics/gstreamer/eme/GStreamerEMEUtilities.h [hunk 1]
```diff
@@ -37,7 +37,7 @@
 #endif
 
 // NOTE: YouTube 2018 EME conformance tests expect this to be >=5s.
```
Comment (eocanha) [inline Source/WebCore/platform/graphics/gstreamer/eme/GStreamerEMEUtilities.h]: Please move this comment to the cpp file, where the constant in now initialized

File: Source/WebKit/Platform/IPC/unix/UnixMessage.h [hunk 2]
```diff
@@ -82,7 +82,6 @@ class UnixMessage {
             memcpy(m_body, other.m_body, m_messageInfo.bodySize());
             m_bodyOwned = true;
             other.m_body = nullptr;
-            other.m_bodyOwned = false;
```
Comment (eocanha) [inline Source/WebKit/Platform/IPC/unix/UnixMessage.h:85]: I don't see what's wrong about trying to clean up m_bodyOwned in the object used as source of the move constructor.

File: Source/WebCore/platform/graphics/gstreamer/eme/WebKitCommonEncryptionDecryptorGStreamer.cpp [hunk 3]
```diff
@@ -180,9 +180,9 @@ static GstCaps* webkitMediaCommonEncryptionDecryptTransformCaps(GstBaseTransform
         }
 
         bool duplicate = false;
-        unsigned size = gst_caps_get_size(transformedCaps);
+        unsigned caps_size = gst_caps_get_size(transformedCaps);
```
Comment (eocanha) [inline Source/WebCore/platform/graphics/gstreamer/eme/WebKitCommonEncryptionDecryptorGStreamer.cpp]: Better use capsSize. WebKit style guide recommends camelCase.

**Review by eocanha — CHANGES_REQUESTED** [review]
Except for the specific comments, the rest of the changes proposed in the two commits look good.

File: Source/WebKit/Platform/IPC/unix/UnixMessage.h (hunk 2 above)
Comment (jacek-manko-red) [reply Source/WebKit/Platform/IPC/unix/UnixMessage.h:85]: Nothing wrong about it, it's not needed though:
        if (other.m_bodyOwned) {
            //...
        } else /*if (... doesn't matter)*/ {
            //...
            other.m_bodyOwned = false; //it already *is* false
        }

I can drop this part if you prefer

**Comment by jacek-manko-red** [comment]
I've updated the branch with suggested changes

File: Source/WebKit/Platform/IPC/unix/UnixMessage.h (hunk 2 above)
Comment (eocanha) [reply Source/WebKit/Platform/IPC/unix/UnixMessage.h:85]: Oh! Now I see after expanding the diff context. Sorry, you're right, please disregard my comment.

**Pull request merged**